php-config/ReCaptcha.config.php

```diff
@@ -0,0 +1,3 @@
+<?php
+\Recaptcha::$sitekey = '6LcfPgsTAAAAAALpupCqLMWwE73GE6_LQDvqC59b';
+\Recaptcha::$secretkey = '6LcfPgsTAAAAAJ6taTfndynhdnpdWcZDc5_U_S0s';
```

Comment:
@RowdyChildren Don't commit your API keys, instead commit a copy of this file with these two lines commented out and INSERT_YOUR_SITE_KEY or some such placeholder, and maybe some comments describing where to get keys